### ISSUE TITLE
How to use hugginface for training:  google-t5/t5-base

### ISSUE THREAD

**Issue by gg22mm** [huggingface/transformers#33232]
### Feature request

How to use hugginface for training / 如何使用huggingface来训练：
 https://github.com/huggingface/transformers/tree/main/examples/pytorch/translation

#What is the format and how do I write it? / 这个格式是怎么样的，怎么写呢？
def batch_collator(data):
    print(data)  #?????????????????????????????????????????????   
    return {
        'pixel_values': torch.stack([x for x in pixel_values]), 
        'labels': torch.tensor([x for x in labels]) 
    }

trainer = Trainer(
    model=model,
    args=training_args,
    data_collator=batch_collator,//这个需要怎么写?
    train_dataset=dataset['train'],    
)

### Motivation

无

### Your contribution

无

我已经试了可以用： https://www.kaggle.com/code/weililong/google-t5-t5-base 
不知道有没有什么坑

**Comment by nbroad1881** [comment]
You can follow the guide [here](https://github.com/huggingface/transformers/tree/main/examples/pytorch/summarization). Even though it says summarization, you can treat it like translation. 

Summarization goes from article --> summary
Translation goes from start language --> end language

**Comment by qgallouedec** [comment]
Hey @gg22mm, sorry for the slow reply, catching up on the backlog. @nbroad1881's answer above covers it: treat the translation setup like the summarization example, just swap source/target language for article/summary.

This tracker is for bugs and feature requests, general "how do I write X" questions are better suited to the forum (https://discuss.huggingface.co/) or the docs, where more people can help. Closing for now, feel free to re-open if you hit a concrete bug.
